blocks/left-rail/left-rail.js

```diff
@@ -13,6 +13,18 @@ export default async function decorate(block) {
   const body = config.body || false;
   const embed = config.embed || false;
 
+  // ---- Move all other sections into a sub-div so we can position the rail alongside it all.
+  const allSections = [...document.querySelectorAll('main .section')];
+  allSections.shift();
```

Comment:
i assume we are performing shift() just to remove hero section inclusion here.. if that is the case, shouldn't we check explicitly? as I persume that not all pages would have hero section?

Reply:
I haven't seen that to be the case.  I'll keep it in mind but reserve the extra work as a bug fix only if warranted.